policyengine_canada/parameters/gov/credits/training/income_threshold.yaml

```
@@ -0,0 +1,24 @@
+description: Income Thresholds for Canada Training Credit. This is a $250 annual credit.
```

I'd suggest also renaming this to `amount.yaml`.

In general we avoid putting numerical values in descriptions.
```suggestion
description: Canada Training Credit amount by income.
```